README.md

```diff
@@ -94,58 +94,242 @@ git fetch upstream-code main
+
+7\. Build the container image from the `/che-code/` directory and push the image to a container registry:
+
+```
+$ docker build -f build/dockerfiles/linux-musl.Dockerfile -t linux-musl-amd64 .
+
+$ docker build -f build/dockerfiles/linux-libc.Dockerfile -t linux-libc-amd64 .
+
+$ export DOCKER_BUILDKIT=1
+
+$ docker build -f build/dockerfiles/assembly.Dockerfile -t vs-code-open-source .
+
+$ docker push <username>/vs-code-open-source:next
+```
+
+8\. Create a `/.che/che-editor.yaml` file in the remote repository that you intend to clone into workspaces. This file must specify the container image of your customized Visual Studio Code - Open Source that is to be pulled for new workspaces.
+
+*Example. `/che-editor.yaml` for the branded Visual Studio Code - Open Source*
+
+```yaml
+inline:
+  schemaVersion: 2.1.0
+  metadata:
+    name: che-code
+  commands:
+    - id: init-container-command
+      apply:
+        component: che-code-injector
+  events:
+    preStart:
+      - init-container-command
+  components:
+    - name: che-code-runtime-description
+      container:
+        image: quay.io/devfile/universal-developer-image:ubi8-latest
+        command:
+          - /checode/entrypoint-volume.sh
+        volumeMounts:
+          - name: checode
+            path: /checode
+        memoryLimit: 2Gi
+        memoryRequest: 256Mi
+        cpuLimit: 500m
+        cpuRequest: 30m
+        endpoints:
+          - name: che-code
+            attributes:
+              type: main
+              cookiesAuthEnabled: true
+              discoverable: false
+              urlRewriteSupported: true
+            targetPort: 3100
+            exposure: public
+            secure: false
+            protocol: https
+          - name: code-redirect-1
+            attributes:
+              discoverable: false
+              urlRewriteSupported: true
+            targetPort: 13131
+            exposure: public
+            protocol: http
+          - name: code-redirect-2
+            attributes:
+              discoverable: false
+              urlRewriteSupported: true
+            targetPort: 13132
+            exposure: public
+            protocol: http
+          - name: code-redirect-3
+            attributes:
+              discoverable: false
+              urlRewriteSupported: true
+            targetPort: 13133
+            exposure: public
+            protocol: http
+      attributes:
+        app.kubernetes.io/component: che-code-runtime
+        app.kubernetes.io/part-of: che-code.eclipse.org
+    - name: checode
+      volume: {}
+    - name: che-code-injector
+      container:
+        image: quay.io/username/vs-code-open-source:next <1>
```

Comment:
I guess `<1>` should be removed:

<img width="538" alt="Screenshot 2022-11-26 at 09 45 33" src="https://user-images.githubusercontent.com/5676062/204078300-a37281c1-c165-4b11-b7b9-bfa0f978d302.png">

Reply:
Oh, thank you for spotting that! Fixing it.